packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -168,7 +168,12 @@ it.skip('Create a directly funded channel between two engines ', async () => {
     turnNum: 0,
   });
 
-  const resultB0 = await b.pushMessage(getPayloadFor(participantB.participantId, preFundA.outbox));
+  const resultB0 = await b.pushMessage(
+    getPayloadFor(
+      participantB.participantId,
+      preFundA.outbox.map(o => o.params)
+    )
```

Comment:
`getPayloadFor` is a test helper. Can it extract the `params` off of an object when needed? The changeset has a lot of noise because of every `o => o.params` callback.

Reply:
https://github.com/statechannels/statechannels/pull/3564/commits/3c1c5ae13281cf595bb1fbd90c60f222a0f356fe